### PR TITLE
Fix electron clipboard action

### PR DESCRIPTION
**Pull request by DhariniJeeva**
## Description

By default electron's security policy blocks all write requests and we need to allow `clipboard-sanitized-write` for the `HDS copy::snippet` copy action to not fail in desktop app

 some relevant links [here](https://www.electronjs.org/docs/latest/tutorial/security#5-handle-session-permission-requests-from-remote-content) and [here](https://www.electronjs.org/docs/latest/api/session#sessetpermissionrequesthandlerhandler)

## Screenshots (if appropriate):
before:
image (6)

after:
Screenshot 2023-10-31 at 5 26 19 PM

## How to Test

- click the port address copy button. You should not see the red icon and no errors in the logs. 
- the screen should look like the `after` image above to indicate that the copy action is successful. 

:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:

- [x] I have added before and after screenshots for UI changes
- ~[ ] I have added JSON response output for API changes~
- [x] I have added steps to reproduce and test for bug fixes in the description
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- ~[ ] I have added tests that prove my fix is effective or that my feature works~

### CODE REVIEW

**Review by dvonanderson — APPROVED** [review]
LGTM

**Review by cameronperera — APPROVED** [review]
LGTM

File: ui/desktop/electron-app/src/index.js [hunk 1]
```diff
@@ -186,6 +186,11 @@ app.on('ready', async () => {
   // per Electronegativity PERMISSION_REQUEST_HANDLER_GLOBAL_CHECK
   ses.setPermissionRequestHandler((webContents, permission, callback) => {
     /* eng-disable PERMISSION_REQUEST_HANDLER_JS_CHECK */
```
Comment (ZedLi) [inline ui/desktop/electron-app/src/index.js:188]: Is this disable still needed?

File: ui/desktop/electron-app/src/index.js [hunk 2]
```diff
@@ -186,6 +186,11 @@ app.on('ready', async () => {
...
+    //we need to allow this for the hds clipboard action to not fail
```
Comment (ZedLi) [inline ui/desktop/electron-app/src/index.js:189]:
```suggestion
    // We need to allow this for native clipboard usage
```

We will likely also need this when fixing the copy paste issue for the embedded terminal in windows so just wanted to adjust the comment

**Review by ZedLi — APPROVED** [review]

**Review by lisbet-alvarez — APPROVED** [review]

**Comment by calcaide** [comment]
@DhariniJeeva sorry I am late. Awesome job on this one!!!

**Pull request merged**